src/lib/client.spec.ts

```diff
@@ -348,3 +348,38 @@ test("supports usage of an async access token getter function", async () => {
   expect(client).toBeInstanceOf(GoogleAdsClient);
   expect(tokenGetterCalled).toEqual("<access-token>");
 });
+
+test("correctly uses logging options", async () => {
+  const client = new GoogleAdsClient({
+    access_token: ACCESS_TOKEN,
+    developer_token: DEVELOPER_TOKEN,
+    login_customer_id: LOGIN_CUSTOMER_ID,
+    logging: {
+      output: "none",
+      verbosity: "info",
+      callback(message) {
+        expect(message.request).toEqual({
+          body: {
+            customerId: "",
+            pageSize: 0,
+            pageToken: "",
+            query: "",
+            returnSummaryRow: false,
+            validateOnly: false,
+          },
+          headers: {
+            authorization: "Bearer ACCESS_TOKEN",
+            "developer-token": "DEVELOPER_TOKEN",
+            "login-customer-id": "LOGIN_CUSTOMER_ID",
+          },
+          method: "/google.ads.googleads.v2.services.GoogleAdsService/Search",
+        });
+      },
+    },
+  });
+
+  const service = client.getService("GoogleAdsService");
+  const request = new SearchGoogleAdsRequest();
+  // It should also throw since we have no auth
```

Comment:
Doesn't look like there is actually an assertion or expecting for logging output here?